Update install instructions (#23)
mymindstorm/matrix-appservice-mumble: README.md

# matrix-appservice-mumble

> A simple Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.

Author: :stuck_out_tongue_winking_eye:

A Matrix to Mumble bridge. It sends messages between bridged rooms and tells you when people join / leave Murmur.

## Install

> ### These instructions are for a development version, please use the instructions located [here](https://github.com/mymindstorm/matrix-appservice-mumble/tree/v0.2.0#install)

Reviewer: Also, could you add this back and have it point to [here, where I have backported this](https://github.com/mymindstorm/matrix-appservice-mumble/tree/0.2.0-install-instructions#installation)?

Author: alrighty x)

    Manually:

    [Download the latest release](https://github.com/mymindstorm/matrix-appservice-mumble/releases) and build

    ```bash
    npm i
    ./build.sh
    ```
2. Configure your homeserver
    1. Use `matrix-appservice-mumble` to generate `mumble-registration.yaml`  
    Replace "http://localhost:port" with the address your homeserver will use to talk with matrix-appservice-mumble.  
    The port matrix-appservice-mumble uses can be set with -p. (Default is 8090)

    ```bash
    matrix-appservice-mumble -r -u "http://localhost:port"
    ```

    2. Copy `mumble-registration.yaml` to your homeserver install directory, e.g. `/etc/matrix-synapse/`

    3. Edit `homeserver.yaml` and add the path to the just generated `mumble-registration.yaml`.

    ```yaml
    # A list of application service config files to use
    app_service_config_files:
    - /etc/matrix-synapse/mumble-registration.yaml
    ```

> 4. Create a file `mumble-config.yaml`

Reviewer: How about this?
```suggestion
4. Create a new file named `mumble-config.yaml` on the server that `matrix-appservice-mumble` is installed on.
```

>     - Look at the [mumble-config.yaml.example](https://github.com/mymindstorm/matrix-appservice-mumble/blob/master/mumble-config.yaml.example) file for an example

Reviewer: ```suggestion
    - Copy the contents of [mumble-config.yaml.example](https://github.com/mymindstorm/matrix-appservice-mumble/blob/master/mumble-config.yaml.example) into `mumble-config.yaml` and change the options as needed.
```

    - `matrixRoom` should be a private room
        
        1. Create a new room (should be invite-only, which is the default)

>         2. Invite `@mumblebot:<your homeserver domain>` to the room. Riot will warn you that the user does not exist, click "Invite anyway"

Reviewer: ```suggestion
        2. Invite `@mumblebot:<your homeserver domain>` to the room. Your client may warn you that the user does not exist, click "Invite anyway"
```

        3. Copy the internal room id of the newly created room to `mumble-config.yaml`

> 5. `matrix-appservice-mumble -c /path/to/mumble-config.yaml -f /path/to/mumble-registration.yaml`

Reviewer: ```suggestion
5. Start the bridge

```bash
$ matrix-appservice-mumble -c /path/to/mumble-config.yaml -f /path/to/mumble-registration.yaml`
```
